### ISSUE TITLE
Nexus 5 - 6.0.1 - No HID attacks

### ISSUE THREAD

**Issue by audibleblink** [offensive-security/kali-nethunter#354]
Device: Nexus 5
ROM: Stock Hammerhead 6.0.1 (MMB29S)
Root: SuperSU-v2.67-20160121175247
Kernel: 3.4.0-gb7558f2

Computer recognizes keyboard. The app "USB Keyboard works". Device has appropriate interfaces

```sh
shell@hammerhead:/ $ ls -la /dev/hid*
crw------- u0_a88   root     241,   0 1972-03-21 00:57 hidg0
crw------- u0_a88   root     241,   1 1972-03-21 00:57 hidg1
```

But ducky scripts don't run. No errors.

**Comment by audibleblink** [comment]
Might be realted to #353

**Comment by audibleblink** [comment]
Fixed with #353
